circuits/move/move.circom

```diff
@@ -136,6 +139,37 @@ template CheckRsrc(N_TL_ATRS, RSRC_IDX, PUBX_IDX, PUBY_IDX, UNOWNED, SYS_BITS) {
         case2, case3]);
 }
 
+/*
+ * The hashes of the old tiles must be included in the merkle root. If so,
+ * this proves that these tiles were computed from prior moves.
+ */
+template CheckMerkleInclusion(N_TL_ATRS, MERKLE_TREE_DEPTH) {
+    signal input root;
+    signal input tFrom[N_TL_ATRS];
+    signal input tFromPathIndices[MERKLE_TREE_DEPTH];
+    signal input tFromPathElements[MERKLE_TREE_DEPTH][1];
+    signal input tTo[N_TL_ATRS];
+    signal input toToPathIndices[MERKLE_TREE_DEPTH];
+    signal input tToPathElements[MERKLE_TREE_DEPTH][1];
+
+    signal hTFrom <== Poseidon(N_TL_ATRS)(tFrom);
+    signal hTTo <== Poseidon(N_TL_ATRS)(tTo);
+
+    signal output out;
+
+    component tFromMerkleProof = MerkleTreeInclusionProof(MERKLE_TREE_DEPTH);
```

Comment:
great work with this function. quick nit here: could you convert the template invocation to anonymous? 

https://docs.circom.io/circom-language/anonymous-components-and-tuples/